app/views/partners/show.html.erb

```diff
@@ -15,9 +15,12 @@
 
     <div class="g g--partner">
       <div class="gi gi__3-5">
-        <% if @partner.short_description %>
+        <% if @partner.summary %>
           <div class="first-ele-lg" property="description">
-            <%= markdown @partner.short_description %>
+            <%= markdown @partner.summary %>
```

Comment:
I was imagining this did not allow markdown so it doesnt look weird on the partners index but that might be confusing for people?

Reply:
I think move this `first-ele-lg` class to just be around the summary field and rename it to something clearer (`.preview__summary`? not sure why its `preview__`!)